app/api/utility.py

```diff
@@ -116,10 +116,10 @@ def create_query(
     query_template = f"""
     {DEFAULT_CONTEXT}
 
-    SELECT ?dataset ?dataset_name ?sub_id
+    SELECT ?dataset ?dataset_name ?sub_id ?file_path ?image_modal
     WHERE {{
     SELECT DISTINCT ?dataset ?dataset_name ?subject ?sub_id ?age ?sex
-    ?diagnosis ?image_modal ?num_sessions ?assessment
+    ?diagnosis ?num_sessions ?assessment ?image_modal ?file_path
```

Comment:
Two comments here:

1. I would spell out `?image_modality` here for clarity. Doesn't add that much typing, and makes it just that little bit clearer we aren't talking about a modal
2. We're now re-using these variable names in other parts of the app (e.g. in `crud.py` to group the results and compute numbers). I think we should add an issue to refactor the SPARQL variable names into constants so we can import and reuse them from there and not have to re-type them correctly everytime.

Reply:
Ah I see. Well not very likely to cause confusion, but modal does have a separate meaning in a UI context https://en.wikipedia.org/wiki/Modal_window

I mostly like the clarity though. But maybe we discuss this separately because it's already established

Reply:
Yea let's do it separately